cli/cluster/cmd/common.go

```diff
@@ -149,10 +149,10 @@ func connectTarantool(uriOpts UriOpts, connOpts connectOpts) (tarantool.Connecto
 		connectorOpts.User = connOpts.Username
 		connectorOpts.Pass = connOpts.Password
 		if connectorOpts.User == "" {
-			connectorOpts.User = os.Getenv(connect.TarantoolUsernameEnv)
+			connectorOpts.User = os.Getenv(libconnect.TarantoolUsernameEnv)
```

Comment:
Why not to use the original package name `connect` instead of `libconnect`?

Reply:
`"github.com/tarantool/tt/cli/connect"` is imported in `tt/cli/cmd/connect.go`. I think it may be confusing to have different `connect`s inside the different files of the same module `cmd`.

Reply:
Removed renaming for modules, except for `cmd` where naming clashes.